pgcli/main.py

```diff
@@ -372,9 +375,17 @@ def run_cli(self):
 
         except EOFError:
             print ('Goodbye!')
-        finally:  # Reset the less opts back to original.
-            logger.debug('Restoring env var LESS to %r.', original_less_opts)
-            os.environ['LESS'] = original_less_opts
+        finally:
+            # Don't persist pgcli less defaults when session is terminated
+            if less_opts_adjusted:
+                os.environ['LESS'] = ''
```

Comment:
It's because `adjust_less_opts` sets defaults only on `if not os.environ.get('LESS'):` - so `less_opts_adjusted` is always false unless they had no LESS options set.